_includes/archive-single.html

```diff
@@ -52,6 +52,10 @@ <h2 class="archive__item-title" itemprop="headline">
     <p class="archive__item-excerpt" itemprop="description"><p>{{ post.excerpt | markdownify | remove: '<p>' | remove: '</p>' }}<strong><a href="{{ base_path }}{{ post.url }}" rel="permalink"> Read more</a></strong></p></p>
     {% endif %}
     
+    {% if post.slidesurl %}
+      <p><a href="{{ post.slidesurl }}"><b>Slides</b></a></p>
+    {% endif %}
```

Comment:
This should be after or adjacent to the link to download the paper, not before the citation.